readme.md

```diff
@@ -15,8 +15,7 @@ You can read the documentation [here](https://nethermindeth.github.io/warp/).
 
 <hr> 
  
-1. Have installed python3.9 and python3.9-dev with the virtualenv ([`venv`](https://docs.python.org/3/library/venv.html)) module in your base env.
-2. You will need [z3](https://github.com/Z3Prover/z3) and [gmp](https://gmplib.org/#DOWNLOAD)
+1. You will need [z3](https://github.com/Z3Prover/z3) and [gmp](https://gmplib.org/#DOWNLOAD)
```

Comment:
Wh did the readme change @GuyMcComb ?

Reply:
I reverted that PR as well, just did the last 4 from the latest. Taking no chances lol.